cog/api/gift.py

```
@@ -0,0 +1,64 @@
+        self.error_msg = None
+        self.headers = {
+            "Authorization": f"Bot {self.api_key}",
+            "Content-Type": "application/json",
+        }
+        try:
+            fetch_usr = self.__new_dm(recipient_id)
+            if "id" in fetch_usr:
+                self.dm_room = fetch_usr["id"]
+            else:
+                self.dm_room = None
+        except Exception as e:
```

Ruff

Do not catch blind exception: `Exception`

(BLE001)